web/sites/default/settings.php

```diff
@@ -1,13 +1,5 @@
 <?php
 
-/**
```

Comment:
I think this might need to stay here. I have run into problems with both settings.dev.php and settings.ts.php expecting variables to exist that don't, because this code grabs them from `$_ENV` and then settings.pantheon.php turns them into vars.

Reply:
Right you are! I forgot that settings.pantheon.php is what does that.